userguide/jails.rst

```diff
@@ -817,44 +772,45 @@ available for a jail.
    | Option       | Description                                                   |
    |              |                                                               |
    +==============+===============================================================+
-   | Edit         | Used to modify the settings described in                      |
-   |              | :numref:`Table %s <jail_overview_tab>`.                       |
-   |              | A jail cannot be edited while it is running. The settings can |
+   | EDIT         | Used to modify the settings described in                      |
+   |              | :ref:`Advanced Jail Creation`.                                |
+   |              | A jail cannot be edited while it is running. The settings     |
    |              | can be viewed, but are read only.                             |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Mount        | Open the :guilabel:`Mount Points` list. Select an existing    |
-   | points       | mount point to :guilabel:`Edit` or click |ui-add| to open     |
-   |              | the :guilabel:`Add Mount Point` screen. A mount point         |
+   | MOUNT        | Select an existing                                            |
+   | POINTS       | mount point to :guilabel:`EDIT` or click                      |
+   |              | :menuselection:`ACTIONS --> Add Mount Point`                  |
+   |              | to create a mount point for the jail. A mount point           |
    |              | gives a jail access to storage located elsewhere on the       |
    |              | system. A jail must be stopped before adding, editing, or     |
-   |              | deleting a :guilabel:`Mount Point`. See                       |
+   |              | deleting a :guilabel:`MOUNT POINT`. See                       |
```

Comment:
Not sure why this one is a guilabel when the previous references are just generic text.  This one probably should be plain text also.